### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by aeh1707**
- Use React documentation.
- Use React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,31 @@
...
 
-# Project Name
+# Discover Crypto
 
-> Description the project.
+> A crypto market app that showcases different top coins with their respective info, built using React and styled with pure CSS. the info is fetched from CoinGecko website.
 
+Check out a demo explanation about this project here [Live Video Demo Explainer](https://www.loom.com/share/0c43fbd5ba9b448ebd97848be08f17c5)
 
 ## Built With
```
Comment (BertrandConxy) [inline README.md]: - [x] Your README is lacking two important sections. First is the `screenshot section` This is where you add the screenshot of your web app to give visitors a quick preview of what your app looks like 😃 you understand that it is very important. The second is `getting started section` How can someone who wants to use your application locally do it? what procedures to follow? what are the prerequisites? how to run the application locally? how to run tests? these are all questions that should be answered in this section so please add those two sections 😉

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hello @aeh1707 ✋, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there! 🏃‍♂️ 🏃 

## Highlights 💪 
- Innovative idea 💯 
- Good UI 💯 
- Live version works well
- correct flow is used
- no linter errors

## Suggested changes

- [ ] Good job so far 👏 Your web app works fine. Your presentation video though is missing some important points. You have not touched on the issue about the piece of code that you are very proud of. This is very important as it showcases your ability to explain technical concepts which will make you outstanding 😉.  So, please record another video and then, include that point 👍 

_Check other  comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (aeh1707) [reply README.md]: @BertrandConxy That's true, thank you for the feedback.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @aeh1707,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**